<?xml version='1.0' encoding='UTF-8'?>
<org.jenkinsci.plugins.workflow.support.storage.SimpleXStreamFlowNodeStorage_-Tag plugin="workflow-support@2.16">
  <node class="org.jenkinsci.plugins.workflow.cps.nodes.StepEndNode" plugin="workflow-cps@2.41">
    <parentIds>
      <string>31</string>
    </parentIds>
    <id>32</id>
    <startId>4</startId>
  </node>
  <actions>
    <org.jenkinsci.plugins.workflow.actions.BodyInvocationAction plugin="workflow-api@2.23.1"/>
    <org.jenkinsci.plugins.workflow.actions.ErrorAction plugin="workflow-api@2.23.1">
      <error class="hudson.AbortException">
        <detailMessage>Deploy_Jar #3 completed with status FAILURE (propagate: false to ignore)</detailMessage>
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.support.steps.build.BuildTriggerListener.onCompleted(BuildTriggerListener.java:53)</trace>
          <trace>hudson.model.listeners.RunListener.fireCompleted(RunListener.java:211)</trace>
          <trace>hudson.model.Run.execute(Run.java:1769)</trace>
          <trace>hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)</trace>
          <trace>hudson.model.ResourceController.execute(ResourceController.java:97)</trace>
          <trace>hudson.model.Executor.run(Executor.java:421)</trace>
        </stackTrace>
        <suppressedExceptions class="java.util.Collections$UnmodifiableRandomAccessList" resolves-to="java.util.Collections$UnmodifiableList">
          <c class="list"/>
          <list reference="../c"/>
        </suppressedExceptions>
      </error>
    </org.jenkinsci.plugins.workflow.actions.ErrorAction>
    <org.jenkinsci.plugins.workflow.actions.TimingAction plugin="workflow-api@2.23.1">
      <startTime>1510526151253</startTime>
    </org.jenkinsci.plugins.workflow.actions.TimingAction>
  </actions>
</org.jenkinsci.plugins.workflow.support.storage.SimpleXStreamFlowNodeStorage_-Tag>